<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>图像区</string>
    </property>
    <widget class="QImageLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>601</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>331</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>控制面板区</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>打开</string>
     </property>
     <widget class="QPushButton" name="btn_open_img">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>打开图片</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_open_dir">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>打开文件夹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_open_list">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>打开文件列表</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>311</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>功能</string>
     </property>
     <widget class="QPushButton" name="btn_start_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始标注</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>撤销</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_end_label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>完成标注</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_prev">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>上一个</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_next">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>下一个</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_view">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>预览</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>圆点半径</string>
      </property>
      <widget class="QComboBox" name="comb_radius">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>进度</string>
     </property>
     <widget class="QProgressBar" name="bar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_prog_point">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_prog_image">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>311</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>说明</string>
     </property>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>141</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;1-'nose',&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;2-'left_eye',3-'right_eye',&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;4-'left_ear' 5-'right_ear',&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;6-'left_shoulder',7-'right_shoulder' 8-'left_elbow',9-'right_elbow',&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;10-'left_wrist' 11-'right_wrist',&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;12-'left_hip' ,13-'right_hip' &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;14-'left_knee',15-'right_knee,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;16-'left_ankle' 17-'right_ankle'&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionOpen_FileList"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionOpen_FileList">
   <property name="text">
    <string>Open FileList</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QImageLabel</class>
   <extends>QLabel</extends>
   <header>qimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
